test-case/check-sof-logger.sh

```diff
@@ -71,7 +71,14 @@ sof_alsa_card_found()
     # note: assumes SOF card names to start with "sof", e.g.
     #   - /proc/asound/sofsoundwire/id
     #   - /proc/asound/sofhdadsp/id
-    test -e /proc/asound/sof*/id
+    # test -e /proc/asound/sof*/id
```

Comment:
@harajend The git commit style is still a bit different than what is the established practise. Please check "git log" history for sof-test to see examples. E.g. there should be some prefix like "check-sof-logger: fix support for multiple SOF cards" on first line. Commit should also explain what is fixed and why, "check-sof-logger.sh fails to detect SOF in case mulitple SOF cards are present in the system. Fix the logic to check the SOF card."